src/pages/Admin/Admin.tsx

```diff
@@ -277,10 +277,16 @@ const EventsComponent: React.FC<VenueDetailsPartProps> = ({ venue }) => {
     },
   ]);
 
-  const events = useSelector((state) => state.firestore.ordered.events);
+  let events = useSelector((state) => state.firestore.ordered.events);
   const [showCreateEventModal, setShowCreateEventModal] = useState(false);
   const [showDeleteEventModal, setShowDeleteEventModal] = useState(false);
   const [editedEvent, setEditedEvent] = useState<WithId<VenueEvent>>();
+  events?.sort((a, b) => a.start_utc_seconds - b.start_utc_seconds);
+  //filter out events that have already finished
+  events = events?.filter(
```

Comment:
Not necessarily a problem but you embedded a commit from the other PR

Reply:
Fixed it on the other PR